### ISSUE TITLE
generated admin credentials not working

### ISSUE THREAD

**Issue by CarstenRuetz** [supergiant/supergiant#673]
HI, I'm trying out supergiant and the generated admin credentials cause the web dashboard to say: Login Error, Error: Invalid Login. 

I followed the guide here: https://supergiant.readme.io/v1.1.0/docs/linux

Then to recreate the admin password:

sudo systemctl stop supergiant-api.service
sudo systemctl stop supergiant-ui.service

// recreate whole database again 
sudo -u postgres psql --command 'DROP DATABASE supergiant;'
sudo -u postgres psql --command 'CREATE DATABASE supergiant;'
sudo -u postgres psql --command 'GRANT ALL PRIVILEGES ON DATABASE supergiant TO supergiant;'

// restart 
sudo systemctl start supergiant-api.service
sudo systemctl start supergiant-ui.service

sudo systemctl status supergiant-api.service 
```
● supergiant-api.service - supergiant-api
   Loaded: loaded (/etc/systemd/system/supergiant-api.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2018-10-06 09:24:25 UTC; 28s ago
 Main PID: 9384 (supergiant-api)
    Tasks: 5 (limit: 1111)
   CGroup: /system.slice/supergiant-api.service
           └─9384 /usr/bin/supergiant-api --config-file /etc/supergiant/config.json

Oct 06 09:24:25 docker-fut systemd[1]: Started supergiant-api.
Oct 06 09:24:26 docker-fut supergiant-api[9384]: time="2018-10-06T09:24:26Z" level=info msg="No Admin detected, creating new and printing credentials:"
Oct 06 09:24:26 docker-fut supergiant-api[9384]:   ( ͡° ͜ʖ ͡°)  USERNAME: admin  PASSWORD: rFWX4hmLR9aETVep
Oct 06 09:24:26 docker-fut supergiant-api[9384]: time="2018-10-06T09:24:26Z" level=info msg="Registering stable HelmRepo"
Oct 06 09:24:26 docker-fut supergiant-api[9384]: Writing log to /var/log/supergiant.log
```

The generated password does produce the error mentioned above.

Any ideas?

**Comment by saheienko** [comment]
Hi, @CarstenRuetz! Thanks for feedback!

I've just tried to install it with the installation script from the docs and then to recreate a user and in both cases got no errors (with `generic/ubuntu1604` vagrant box).

Could you try to access the supergiant-api directly and review the error:
```
curl -XPOST -d'{"user":{"username":"admin","password":"YOUR_PASSWORD"}}' localhost/api/v0/sessions
```

**Comment by CarstenRuetz** [comment]
Hi @saheienko, thanks for the fast reply!

The output of
`curl -XPOST -d'{"user":{"username":"admin","password":"0qt3kFgJo9awBAem"}}' localhost:8080/api/v0/sessions`

```
{
  "id": "GUI345ST8AObWNeYgBCjklmnopJrstMR",
  "user_id": 1,
  "created_at": "2018-10-10T15:54:08.997567841Z",
  "user": null
}
```

I interpret this as "I'm logged in" I guess...
So I tried the web interface again: and it worked. Unexpected...

I tried to reproduce the same behaviour described above. Result: I can login with the provided credentials. I'm surprised. First of all now: sorry for bothering you with this.

I cannot even limit the reasons any more: it's the same machine, no changes as far as I remember. However I cannot rule out that s.th. was updated. 

So, solved for me. But weird anyway.
